TERRA_QUICKSTART.md

```diff
@@ -0,0 +1,47 @@
+# Quickstart for Terra Devs
+
+This document is intended for folks who need to do basic JobManager maintenance
+in support of its use in Terra. 
+
+## Running JobManager Locally
+
+These are the minimal steps you need to take to get JobManager running locally in 
+Docker, showing state from a local Cromwell instance. It combines steps from the 
+main README and Cromwell server README.
+
+### Check out and configure JobManager
+
+```
+$ git clone https://github.com/DataBiosphere/job-manager.git
+$ cd job-manager
+$ ln -sf cromwell-instance-compose.yml docker-compose.yml
+```
+
+### Set CROMWELL_URL env var
+
+JobManager relies on a `CROMWELL_URL` environment variable to find Cromwell.
+Set this to `http://IP_ADDR:8000/api/workflows/v1` where `IP_ADDR` is your machine's 
+IP address. We need the broadcast address, if we use `0.0.0.0` or `localhost` Docker
+will try to connect to itself.
+
+For example:
+```
+$ ifconfig | grep inet | grep broadcast
```

Comment:
thanks for the example. I didn't know what "broadcast" meant here, but it didn't matter.

Reply:
I'll admit that network stuff is a weak point for me - is there a different way to say this that would have made it clearer to you? External IP address?

Reply:
No, I don’t have a better suggestion. I’m also not very knowledgeable about network stuff! I was just calling out what I saw as a good thing (that I, as the reader, didn’t have to know the specifics, because there’s an example).